src/shared/GitHub/GitHubHostProvider.cs

```diff
@@ -183,6 +193,53 @@ public async Task<ICredential> GetCredentialAsync(InputArguments input)
             return credential;
         }
 
+        private bool FilterAccounts(Uri remoteUri, IEnumerable<string> wwwAuth, ref IList<string> accounts)
+        {
+            if (!IsGitHubDotCom(remoteUri))
+            {
+                _context.Trace.WriteLine("No account filtering outside of GitHub.com.");
```

Comment:
Should everything below be skipped if remoteUri is not a github.com by returning a false? Like, in case there is a link to the GHES.